core/src/main/java/org/mindtrails/domain/Action/Action.java

```diff
@@ -0,0 +1,25 @@
+package org.mindtrails.domain.Action;
+
+import lombok.Data;
+import org.mindtrails.domain.data.DoNotDelete;
+import org.mindtrails.domain.data.Exportable;
+import org.mindtrails.domain.questionnaire.LinkedQuestionnaireData;
+
+import javax.persistence.Entity;
+import javax.persistence.Lob;
+import javax.persistence.Table;
+import java.util.Date;
+
+@Entity
+@Table(name = "Action")
+@Data
+@Exportable
+@DoNotDelete
+public class Action extends LinkedQuestionnaireData {

```

Comment:
Good point - let me double check and look into this. My reason for doing this was to ensure that questionnaires are associated with action sequences, so that later on, we could do a questionnaire-by-questionnaire analysis of average latency per questionnaire, largest latency per questionnaire, etc. But you're right - I should check the other Logs and follow their structure.